### DISCUSSION TITLE
Notice: Updated specification for pattern-matching

### DISCUSSION THREAD

**Discussion by gafter** [dotnet/csharplang#2090] [General]
I have updated https://github.com/dotnet/csharplang/blob/master/proposals/patterns.md to reflect the latest on our plans (and prototype) for pattern-matching in C# 8.

**Comment by Unknown6656** [comment]
The url "http://lampwww.epfl.ch/~emir/written/MatchingObjectsWithPatterns-TR.pdf" has been 403'ed.
(It is the Scala-link in the first paragraph).

**Comment by gafter** [comment]
Fixed.

**Comment by bbarry** [comment]
current link: (2019-08-01) https://github.com/dotnet/csharplang/blob/master/proposals/csharp-8.0/patterns.md
